stripe-ruby-mock.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'stripe', '~> 1.31'
+  gem.add_dependency 'stripe', '~> 2.8'
```

Comment:
existing code works fine with stripe 1.58. With 2.0 it totally broken. The main problem in ```ruby
Stripe::StripeClient.active_client.method(:request)
```
you fixed it. I want to make a release with 1.31-1.58 stripe gem for old projects and 2.0 and newest for new projects. You set 2.1 as minimum and it means that people who use 2.0 stripe can't use any of stripe-ruby-mock release.